consensus/processor.go

```diff
@@ -504,26 +516,26 @@ type ApplyMessagesResponse struct {
 	TemporaryErrors []error
 }
 
-// ApplyMessagesAndPayRewards begins by paying the block mining reward to the miner. It then applies messages to a state tree.
+// ApplyMessagesAndPayRewards begins by paying the block mining reward to the miner's owner. It then applies messages to a state tree.
 // It returns an ApplyMessagesResponse which wraps the results of message application,
 // groupings of messages with permanent failures, temporary failures, and
 // successes, and the permanent and temporary errors raised during application.
 // ApplyMessages will return an error iff a fault message occurs.
 // Precondition: signatures of messages are checked by the caller.
-func (p *DefaultProcessor) ApplyMessagesAndPayRewards(ctx context.Context, st state.Tree, vms vm.StorageMap, messages []*types.SignedMessage, minerAddr address.Address, bh *types.BlockHeight, ancestors []types.TipSet) (ApplyMessagesResponse, error) {
+func (p *DefaultProcessor) ApplyMessagesAndPayRewards(ctx context.Context, st state.Tree, vms vm.StorageMap, messages []*types.SignedMessage, minerOwnerAddr address.Address, bh *types.BlockHeight, ancestors []types.TipSet) (ApplyMessagesResponse, error) {
 	var emptyRet ApplyMessagesResponse
 	var ret ApplyMessagesResponse
 
-	// transfer block reward to miner from network address.
-	if err := p.blockRewarder.BlockReward(ctx, st, minerAddr); err != nil {
+	// transfer block reward to miner's owner from network address.
+	if err := p.blockRewarder.BlockReward(ctx, st, minerOwnerAddr); err != nil {
```

Comment:
i wish we had had a policy of only returning fault errors at the level of apply* so that it was easier to tell by inspection if fault errors are being returned. as it is we have to go look in each implementation we call and verify it. no action required, just musing.